package.json

```diff
@@ -7,6 +7,7 @@
     "dev": "cross-env NODE_ENV=development nodemon server.js",
     "lint": "eslint .",
     "lint-fix": "eslint . --fix",
+    "test-config": "node local-test.js",
```

Comment:
Now there will be no local config check for running development server.
Only test-config variable will be responsible for checking local configuration.